### DISCUSSION TITLE
error InvalidGitRepositoryError(epath)

### DISCUSSION THREAD

**Discussion by wind91725** [gitpython-developers/GitPython#1155] [Q&A] [answered]
my GitPython==3.1.3，my error below，how can i solve the issus？please help me，thank you!
  File "tools/run.py", line 88, in <module>
    run()
  File "tools/run.py", line 75, in run
    trainer.load()
  File "/data/home/share1/pythia/pythia/trainers/base_trainer.py", line 49, in load
    self.load_extras()
  File "/data/home/share1/pythia/pythia/trainers/base_trainer.py", line 146, in load_extras
    self.checkpoint = Checkpoint(self)
  File "/data/home/share1/pythia/pythia/utils/checkpoint.py", line 51, in __init__
    self.repo = git.Repo(self.repo_path)
  File "/data/home/share1/36/anaconda3/envs/pythia/lib/python3.6/site-packages/git/repo/base.py", line 181, in __init__
    raise InvalidGitRepositoryError(epath)
git.exc.InvalidGitRepositoryError: /data/home/share1/pythia

**Comment by Byron** [comment, marked as answer]
I think without information on what you are trying to do there will not be much anyone can do for you. Thus I am closing the issue and consider it Q&A.

As a possible way to help understand what the problem is, you could enter the path ` /data/home/share1/pythia` yourself and execute git inside of it. Maybe the git command also won't be able to recognise it as repository. From there one could find it why that is [that causes no .git dir to be found](https://github.com/gitpython-developers/GitPython/blob/master/git/repo/base.py#L181:L181).

**Comment by zh57398** [comment]
> my GitPython==3.1.3，my error below，how can i solve the issus？please help me，thank you!
> File "tools/run.py", line 88, in
> run()
> File "tools/run.py", line 75, in run
> trainer.load()
> File "/data/home/share1/pythia/pythia/trainers/base_trainer.py", line 49, in load
> self.load_extras()
> File "/data/home/share1/pythia/pythia/trainers/base_trainer.py", line 146, in load_extras
> self.checkpoint = Checkpoint(self)
> File "/data/home/share1/pythia/pythia/utils/checkpoint.py", line 51, in **init**
> self.repo = git.Repo(self.repo_path)
> File "/data/home/share1/36/anaconda3/envs/pythia/lib/python3.6/site-packages/git/repo/base.py", line 181, in **init**
> raise InvalidGitRepositoryError(epath)
> git.exc.InvalidGitRepositoryError: /data/home/share1/pythia

Have you solved the problem? I also encountered similar problems, looking forward to your reply, thank you very much.
